stdlib/bytes.gr

```diff
@@ -7,6 +7,7 @@ import Conv from "runtime/unsafe/conv"
 import { tagSimpleNumber, allocateBytes, allocateString } from "runtime/dataStructures"
 import Exception from "runtime/exception"
 import Int32 from "int32"
+import { coerceNumberToWasmI32 as numberToWasmI32 } from "runtime/numbers"
```

Comment:
Can we just rename the `numberToWasmI32` to `coerceNumberToWasmI32` in this file?

Reply:
Renamed.